include/ginkgo/core/base/math.hpp

```diff
@@ -716,8 +754,8 @@ isfinite(const T &value)
  *         they are neither +/- infinity nor NaN.
  */
 template <typename T>
-GKO_INLINE GKO_ATTRIBUTES xstd::enable_if_t<is_complex_s<T>::value, bool>
-isfinite(const T &value)
+GKO_INLINE GKO_ATTRIBUTES xstd::enable_if_t<is_complex<T>(), bool> isfinite(
```

Comment:
could you revert this one? (is_complex<T>() -> is_complex_s<T>::value)
This is related to Windows compile. Microsoft Visual Studio can not understand it.

Reply:
Also need to change others